src/patternfly/components/Touchspin/touchspin.scss

```diff
@@ -0,0 +1,55 @@
+.pf-c-touchspin {
+  // main
+  --pf-c-touchspin__main--m-active--c-form-control--Color: var(--pf-global--Color--100);
+
+  // units
+  --pf-c-touchspin__main__units--MarginLeft: var(--pf-global--spacer--sm);
+  --pf-c-touchspin__units__main--MarginLeft: var(--pf-global--spacer--sm);
+
+  // control
+  --pf-c-touchspin__control--m-threshold--c-button--Color: var(--pf-global--disabled-color--100);
+
+  // icon
+  --pf-c-touchspin__control-icon--FontSize: var(--pf-global--FontSize--xs);
+
+  // form control
+  --pf-c-touchspin--c-form-control--Color: var(--pf-global--disabled-color--100);
+  --pf-c-touchspin--c-form-control--MaxWidth: 6ch;
+
+  display: flex;
+  align-items: center;
+
+  .pf-c-form-control {
+    display: inline;
+    max-width: var(--pf-c-touchspin--c-form-control--MaxWidth);
+    color: var(--pf-c-touchspin--c-form-control--Color);
+    text-align: right;
+  }
+}
+
+.pf-c-touchspin__main {
+  &.pf-m-active {
+    --pf-c-touchspin--c-form-control--Color: var(--pf-c-touchspin__main--m-active--c-form-control--Color);
+  }
+}
+
+.pf-c-touchspin__main + .pf-c-touchspin__units {
+  margin-left: var(--pf-c-touchspin__main__units--MarginLeft);
+}
+
+.pf-c-touchspin__units + .pf-c-touchspin__main {
+  margin-left: var(--pf-c-touchspin__main__units--MarginLeft);
+}
+
+.pf-c-touchspin__control {
+  &.pf-m-threshold {
+    .pf-c-button {
```

Comment:
can you just disable the button? React can do that once the threshold is met, I don't think we need a class for it.

Reply:
the reason i added the modifier class was because adding disabled on the button gives it a background color, and removes all the borders from the button. I thought this might be leaner instead of adding back styles, but let me know what you think!

Reply:
Ah!  I'm not sure why we would need to deviate from the default design here since the button is effectively disabled. I would be in favor of just letting the button be disabled so it's consistent with disabled buttons in the rest of patternfly.

Reply:
@mcarrano @maryshak1996 wdyt? Do the disabled form control buttons in the touchspin need to be presented differently than regular disabled form control buttons?

@mcarrano also made the comment that he could also see plain buttons used in this component, which wouldn't apply a background when disabled.

Reply:
This is how it would look, what do you think @maryshak1996 @mcarrano 

<img width="505" alt="Screen Shot 2020-10-20 at 5 21 01 PM" src="https://user-images.githubusercontent.com/20118816/96645576-c3236c00-12f8-11eb-827e-00305abec12b.png">

Reply:
I like it @christiemolloy . I think this makes it very clear that the buttons are disabled and is consistent with how we handle disabled button states elsewhere.

Reply:
Do you also think we need an example with the entire control disabled (i.e. buttons and the field)?

Reply:
yep good idea ill add that!